test/gui/widgets/test_changelog_widget.py

```diff
@@ -1,33 +1,74 @@
 from __future__ import annotations
 
-from typing import cast
+from typing import TYPE_CHECKING, cast
 
 from PySide6 import QtWidgets
 
 from randovania.gui.widgets.changelog_widget import ChangeLogWidget
+from randovania.gui.widgets.delayed_text_label import DelayedTextLabel
 
+if TYPE_CHECKING:
+    from pytestqt.qtbot import QtBot
 
-def test_create(skip_qtbot):
+IMG_LINK = "https://user-images.githubusercontent.com/884928/42738464-8a68e82a-885a-11e8-8b05-a2af0b8ad9e9.png"
+
+IMG_TAG = f"![image]({IMG_LINK})"
+
+CUSTOM_LOGS = {
+    "1.0": "Foo",
+    "2.0": "Bar",
+    "3.0": f"### Testing ^_^\n{IMG_TAG}\nText after image",
+    "4.0": f"### Get ready for 2 images!!!\nFirst:\n{IMG_TAG}\nSecond:\n{IMG_TAG}\nblah blah blah",
+}
+
+CUSTOM_LOGS_KEYS = list(CUSTOM_LOGS.keys())
+
+EXPECTED_VERSIONS_COUNT = len(CUSTOM_LOGS_KEYS)
+
+
+async def test_create(skip_qtbot: QtBot, mocker):
     # Setup
-    widget = ChangeLogWidget(
-        {
-            "1.0": "Foo",
-            "2.0": "Bar",
-        }
-    )
+    mocked_fetch = mocker.patch("randovania.gui.widgets.changelog_widget.ChangeLogWidget.startFetchingData")
+    widget = ChangeLogWidget(CUSTOM_LOGS)
     skip_qtbot.addWidget(widget)
 
     # Assert
-    assert widget.select_version.count() == 2
-    assert widget.select_version.itemText(0) == "1.0"
-    assert widget.select_version.itemText(1) == "2.0"
+    with skip_qtbot.waitSignal(widget.doneFetchingData):
+        mocked_fetch.emit.assert_called()
+        await widget.setup_lables()
+        assert widget.select_version.count() == EXPECTED_VERSIONS_COUNT
+
+    for i in range(1, EXPECTED_VERSIONS_COUNT + 1):
+        assert widget.select_version.itemText(i - 1) == CUSTOM_LOGS_KEYS[i - 1]
 
     widget.select_version.setCurrentIndex(0)
     qScroll0 = cast(QtWidgets.QScrollArea, widget.changelog.currentWidget())
-    qLabel0 = cast(QtWidgets.QLabel, qScroll0.widget())
+    qFrame0 = cast(QtWidgets.QFrame, qScroll0.widget())
+    qLabel0 = cast(DelayedTextLabel, qFrame0.findChild(DelayedTextLabel))
```

Comment:
snake_case for all the `qBlaX` variables.
`qScroll0` was already there before but would be nice to fix that